### DISCUSSION TITLE
Server side rendering with remote location

### DISCUSSION THREAD

**Discussion by laurentiustroia** [module-federation/module-federation-examples#1249] [Q&A]
Hi, I try use module federation on server side for rendering app, I start play with example from `server-side-rendering` but If I replace `remotePath` with `const remotePath = "http://localhost:3002/static/container.js";` in `server.base.js` ( which is identical with `client.base.js` config) I go this error:

```js
- /module-federation-examples/server-side-rendering/website1/buildServer/main.js
while loading "./SomeComponent" from 37550

```

How should deal with module federation when all module will be hosted on each server and there aren't any relative/absolute path for that ?

**Comment by ScriptedAlchemy** [comment]
I'm working on it. Dm me on twitter. Is this for vanilla node? Not nextjs or anything else
